static/panes/cfg-view.ts

```diff
@@ -190,6 +192,27 @@ export class Cfg extends Pane<CfgState> {
         this.exportPNGButton = this.domRoot.find('.export-png').first();
         this.estimatedPNGSize = unwrap(this.exportPNGButton[0].querySelector('.estimated-export-size'));
         this.exportSVGButton = this.domRoot.find('.export-svg').first();
+
+        // create a fake .graph-container .graph .block-container where we can compute block dimensions
```

Comment:
can you please refactor this into a new function

Reply:
Thank you that's a good idea